test/lit.cfg

```diff
@@ -1536,7 +1536,7 @@ config.substitutions.append(('%llvm-profdata', config.llvm_profdata))
 config.substitutions.append(('%llvm-cov', config.llvm_cov))
 
 config.substitutions.append(('%FileCheck',
-                             '%r %r --sanitize BUILD_DIR=%r --sanitize SOURCE_DIR=%r --use-filecheck %r %s' % (
+                             '%r %r --sanitize BUILD_DIR=%r --sanitize SOURCE_DIR=%r --use-filecheck %r --dump-input fail %s' % (
```

Comment:
Does the path sanitizing FileCheck let through the argument and transform it to have an '='?

Reply:
It lets the argument through. I don't think it transforms it. The docs seem to suggest that it's acceptable with or without the `=`: https://llvm.org/docs/CommandGuide/FileCheck.html. At any rate, it does work.